src/DotNetLightning.Core/Crypto/RevocationSet.fs

```diff
@@ -25,7 +25,30 @@ type InsertRevocationKeyError =
 type RevocationSet private (keys: list<CommitmentNumber * RevocationKey>) =
     new() = RevocationSet(List.empty)
 
-    member private this.Keys = keys
+    member this.Keys = keys
```

Comment:
can you change to internal instead of public?

Reply:
No, part of the point of this PR is to expose this member so that we can access it and serialize it from geewallet.